nixos/tests/systemd-initrd.nix

```diff
@@ -0,0 +1,24 @@
+import ./make-test-python.nix ({ lib, pkgs, ... }: {
```

Comment:
Maybe call the entire test `systemd-initrd-simple` or something like this so it's more consistent when adding more tests in the future